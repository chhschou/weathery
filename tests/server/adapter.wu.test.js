const data = require('../data/wu.js')

test('wu conditions adaptor works', () => {
  const actual = data.getConditions()
  expect(actual).toMatchSnapshot()
})
