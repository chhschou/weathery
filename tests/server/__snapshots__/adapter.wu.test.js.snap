// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`wu conditions adaptor works 1`] = `
Object {
  "feelslikeC": "13.1",
  "feelslikeF": "55.5",
  "iconUrl": "http://icons.wxug.com/i/c/k/nt_mostlycloudy.gif",
  "tempC": 13.1,
  "tempF": 55.5,
  "text": "Mostly Cloudy",
  "timeStamp": "1504869815",
}
`;
